### PR TITLE
T1523

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/189d2fe3366de31/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/189d2fe3366de31/output.txt

Total script time: 1.33 mins

File: src/ui/page-element.js [hunk 1]
```diff
@@ -144,13 +144,15 @@ define( [ "core/logger", "core/eventmanager", "util/dragndrop", "ui/position-tra
...
+    this.highlightElement = _highlightElement;
+
     Object.defineProperties( this, {
       element: {
         enumerable: true,
         get: function(){
           return _element;
         }
-      }
+      },
```
Comment (ScottDowne) [inline src/ui/page-element.js]: Can we remove this comma?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/29068bb69d9b12b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/95d61b89e68f14b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/29068bb69d9b12b/output.txt

Total script time: 0.26 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/95d61b89e68f14b/output.txt

Total script time: 0.56 mins

Server is running on http://pr491.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/8786d8b7342afee/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8786d8b7342afee/output.txt

Total script time: 0.59 mins

Server is running on http://pr491.butyr.org/templates/test.html

**Pull request merged**